### DISCUSSION TITLE
Sharing a child across multiple parents in Flows

### DISCUSSION THREAD

**Discussion by manuganji** [taskforcesh/bullmq#2480] [Q&A] [answered]
Hi, am I correct to assume that by using the same job id, I can have a child present in multiple parent jobs? My concern is to avoid the same job being processed more than once while making the result available downstream in more than one paths.

**Comment by manuganji** [comment]
I think #2416 is also about the same concern.

**Reply by manast** [reply, marked as answer]
It is currently not possible to have the same child on more than one parent.
